app/src/main/java/com/concordium/wallet/onboarding/ui/terms/TermsAndConditionsFragment.kt

```diff
@@ -0,0 +1,112 @@
+package com.concordium.wallet.onboarding.ui.terms
+
+import android.content.Intent
+import android.os.Bundle
+import android.text.util.Linkify
+import android.view.View
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.lifecycleScope
+import androidx.lifecycle.repeatOnLifecycle
+import com.concordium.wallet.App
+import com.concordium.wallet.R
+import com.concordium.wallet.databinding.FragmentTermsAndConditionsBinding
+import com.concordium.wallet.ui.auth.login.AuthLoginActivity
+import com.concordium.wallet.ui.base.BaseBindingFragment
+import com.concordium.wallet.ui.intro.introstart.IntroStartActivity
+import com.google.android.material.snackbar.Snackbar
+import com.walletconnect.util.Empty
+import kotlinx.coroutines.launch
+import org.koin.androidx.viewmodel.ext.android.activityViewModel
+import java.util.regex.Pattern
+
+internal const val TERMS_AND_CONDITIONS_WEB_LINK =
+    "https://developer.concordium.software/en/mainnet/net/resources/terms-and-conditions.html"
```

Comment:
This should not be hardcoded. It should instead use the `url` field from `TermsAndConditionsItem` which, as far as I can tell, isn't currently being used for anything?

Reply:
fixed